### PR TITLE
Updated trolcommander version to 0.9.7 and github URL

### PR DESCRIPTION
**Pull request by rajivnarayan**
Updated to the current version
Recent development and releases seem to have switched to new repo, so I've updated the URL

### CODE REVIEW

**Comment by jawshooah** [comment]
Neither the commit message nor the PR title mention the name of the updated cask. Please amend both.

**Comment by vitorgalvao** [comment]
If you don’t know how to amend, follow [this](http://stackoverflow.com/a/179147/1661012).

**Comment by fanquake** [comment]
Looks like you've introduced extra commits when changing the commit message. A maintainer will manually fix and merge this PR.

File: Casks/trolcommander.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'trolcommander' do
-  version '0.9.6'
-  sha256 '2378192a989c83e73ffad84a93b281ba2d0f8aeaac5763a454c3d66308423904'
+  version '0.9.7'
+  sha256 'd67cdf6c195e3885daf7d22243e4596b39dbbeb45f67c465ed3f34b2468110bd'
 
   # github.com is the official download host per the vendor homepage
```
Comment (adidalal) [inline Casks/trolcommander.rb]: needs new `url` comment based on https://github.com/caskroom/homebrew-cask/pull/18051

File: Casks/trolcommander.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-  url "https://github.com/trol73/trol-commander-dist/blob/master/#{version}/trolcommander-#{version.gsub('.', '_')}.app.tar.gz?raw=true"
+  url "https://github.com/trol73/mucommander/releases/download/v#{version}/trolcommander-#{version.gsub('.', '_')}.app.tar.gz?raw=true"
```
Comment (adidalal) [inline Casks/trolcommander.rb]: version.dots_to_underscores

**Comment by victorpopkov** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 3e9ead8c7315a488e9147039d2fb8d698258f55c. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**